Jenkinsfile

```diff
@@ -123,59 +123,31 @@ node('vetsgov-general-purpose') {
       // Set timeout of 60 minutes for integration tests
       timeout(60) {
         try {
-          if (commonStages.IS_PROD_BRANCH && commonStages.VAGOV_BUILDTYPES.contains('vagovprod')) {
```

Comment:
Turns out there was no difference between the branches of this conditional...

Reply:
I presume the answer is no, but would this still execute if not `PROD_BRANCH`?

Reply:
This particular conditional? No, but the else block would've been executed, but it seems identical to this if block anyway.